corehq/util/datadog/utils.py

```diff
@@ -82,3 +82,20 @@ def get_url_group(url):
 def update_datadog_metrics(metrics):
     for metric, value in metrics.items():
         statsd.gauge(metric, value)
+
+
+def bucket_value(value, buckets, unit=None):
```

Comment:
might be nice to wrap this in a factory function so it could be used like this:
```py
_get_time_bucket = make_bucket_func([5, 20, 60, 120], 's')

class RestoreConfig(object):
    def _record_timing(self, duration):
        bucket = _get_time_bucket(duration)
        ...
```